utils/version.py

```diff
@@ -488,6 +488,7 @@ def __cmp__(self, other):
 
 LOWEST = LooseVersion(oldest=True)
 LATEST = LooseVersion(latest=True)
+UPSTREAM = LATEST
```

Comment:
We're going to have multiple upstreams soon, which will bring in a distinction between upstream and "latest". This solution is fine for now, but it essentially means that we should change all references from LATEST to UPSTREAM.

All that is to say that you should probably not do this in this PR.